starter-code/public/css/styles.css

```diff
@@ -0,0 +1,37 @@
+body {
+    background-color: brown;
+}
+
+.col-xs-4 {
```

Comment:
Please don't make changes to default Bootstrap classes. This way, you're overwriting Bootstrap and potentially this may lead to losing its responsiveness.